community-operators/federation/federation/0.0.5/federation.v0.0.5.clusterserviceversion.yaml

```diff
@@ -0,0 +1,299 @@
+          - events
+          - configmaps
+          - secrets
+          verbs:
+          - "*"
+        - apiGroups:
+          - apps
+          resources:
+          - deployments
+          - daemonsets
+          - replicasets
+          - statefulsets
+          verbs:
+          - "*"
+      # TODO(font): use statefulset
+      deployments:
+      - name: federation-controller-manager
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: federation-controller-manager
+          template:
+            metadata:
+              labels:
+                app: federation-controller-manager
+            spec:
+              containers:
+                - name: controller-manager
+                  image: quay.io/pmorie/origin-federation-controller:v4.0.0
+                  resources:
+                    limits:
+                      cpu: 100m
+                      memory: 128Mi
+                    requests:
+                      cpu: 100m
+                      memory: 64Mi
+                  command:
+                    - /root/controller-manager
+                  args:
+                    - --federation-namespace=$(FEDERATION_NAMESPACE)
+                    - --install-crds=false
+                    - --limited-scope=true
+                    - --registry-namespace=$(CLUSTER_REGISTRY_NAMESPACE)
+                    - -v=5
+                    - --logtostderr
+                  imagePullPolicy: Always
+                  env:
+                  - name: FEDERATION_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+                  - name: CLUSTER_REGISTRY_NAMESPACE
+                    valueFrom:
+                      fieldRef:
+                        fieldPath: metadata.namespace
+              restartPolicy: Always
+              terminationGracePeriodSeconds: 5
+              serviceAccountName: federation-controller-manager
+              serviceAccount: federation-controller-manager
+  customresourcedefinitions:
+    owned:
+        # TODO(font): Move Cluster CRD to required once OLM supports CSVs
+        # without a deployment.
+      - description: Represents an instance of a Cluster Registry
+        displayName: Cluster Registry Application
+        kind: Cluster
+        name: clusters.clusterregistry.k8s.io
+        version: v1alpha1
+      - description: Represents a Domain name and nameserver for use with multicluster DNS.
+        displayName: Domain
+        kind: Domain
+        name: domains.multiclusterdns.federation.k8s.io
+        version: v1alpha1
+      - description: Represents an instance of a FederatedCluster resource
+        displayName: FederatedCluster Resource
+        kind: FederatedCluster
+        name: federatedclusters.core.federation.k8s.io
+        version: v1alpha1
+      # - description: Represents an instance of a FederatedConfigMap resource
```

Comment:
These are CRDs that used to be installed by the CSV. They are now generated by the user instead of staticaly defined. I commented them out because we can't ship them in the CSV without also being able to install instances of another CR, which I understand not to be supported currently by the marketplace.

So, TLDR; I will remove the commented-out resources for now.